test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -69,5 +64,33 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains("/Views/ErrorFromViewImports/_ViewImports.cshtml", content);
             Assert.Contains(expectedMessage, content);
         }
+
+        [Fact]
+        public async Task RuntimeErrorAreListedByErrorPageMiddleware()
+        {
+            // The desktop CLR does not correctly read the stack trace from portable PDBs. However generating full pdbs
+            // is only supported on machines with CLSID_CorSymWriter available. On desktop, we'll skip this test on 
+            // machines without this component.
```

Comment:
We could fix that by referencing the DiaSymReader but then it'll get deployed with every ASP.NET site